benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/workload/users/UserValuesYamlProtocol.scala

```diff
@@ -0,0 +1,162 @@
+package cloud.benchflow.dsl.definition.configuration.goal.explorationspace.workload.users
+
+import cloud.benchflow.dsl.definition.configuration.goal.explorationspace.ExplorationSpaceYamlProtocol
+import cloud.benchflow.dsl.definition.configuration.goal.explorationspace.step.RangeWithStep
+import cloud.benchflow.dsl.definition.errorhandling.BenchFlowDeserializationException
+import cloud.benchflow.dsl.definition.errorhandling.YamlErrorHandler.{ deserializationHandler, unsupportedReadOperation, unsupportedWriteOperation }
+import net.jcazevedo.moultingyaml.{ DefaultYamlProtocol, YamlFormat, YamlObject, YamlString, YamlValue, _ }
+
+import scala.util.{ Failure, Success, Try }
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-23
+ */
+object UserValuesYamlProtocol extends DefaultYamlProtocol {
```

Comment:
I did not do that because for each type the allowed ranges might be different. I think it is better to do this refactoring once there is a need.